plugins/woocommerce/src/Internal/Admin/Orders/Edit.php

```diff
@@ -233,6 +232,10 @@ public function handle_order_update() {
 		 */
 		do_action( 'woocommerce_process_shop_order_meta', $this->order->get_id(), $this->order );
 
+		if ( wc_get_container()->get( CustomOrdersTableController::class )->custom_orders_table_usage_is_enabled() ) {
```

Comment:
This check shouldn't be necessary as `Edit` is only used within an HPOS codepath. I believe it should be safe to assume that it is enabled. Context:

https://github.com/woocommerce/woocommerce/blob/93b6f1b8ce5b27064ef9745cf4f49026365b114e/plugins/woocommerce/src/Internal/Admin/Orders/PageController.php#L349 and https://github.com/woocommerce/woocommerce/blob/93b6f1b8ce5b27064ef9745cf4f49026365b114e/plugins/woocommerce/includes/admin/class-wc-admin-menus.php#L340-L341)

Also, that line is producing a fatal error because the `CustomOrdersTableController` can't be resolved.

Reply:
Argh. I added the HPOS check in the last moment before committing and didn't think that an import would be needed.

Anyway, you are right, the check isn't needed. I've removed it.